docs/config.md

```diff
@@ -2,20 +2,16 @@
 
 To use TypingDNA with WSO2 Identity Server, first you need to configure the authenticator with WSO2 Identity Server. The following topics provide instructions on how to configure the TypingDNA with WSO2 Identity Server as a risk-based authentication (RBA) option:
```

Comment:
General comment: Images are not aligned with the numbering.
<img width="1114" alt="Screenshot 2021-11-12 at 13 14 58" src="https://user-images.githubusercontent.com/21237558/141429535-8fb5ff36-7fb8-4287-bb74-e576f7026696.png">

Reply:
Addressed with https://github.com/wso2-extensions/identity-conditional-auth-typingdna/pull/15/commits/809fb935c083c4e9bed561daf6bd729580a9e074